app/components/Identicon/Identicon.tsx

```diff
@@ -6,9 +7,11 @@ type IdenticonProps = {
   seed: string
   type: "user" | "account"
   alt?: string
+  size?: MantineNumberSize
+  avatar?: boolean
```

Comment:
this doesnt seem to be user

Reply:
avatar prop is used under `OrganizationalDrawer`